internal/pkg/scalable/statefulsets.go

```diff
@@ -73,7 +73,7 @@ func (s *statefulSet) ScaleUp() error {
 	return nil
 }
 
-// ScaleDown downscale the resource when the downscale period starts
+// ScaleDown downscale the resource
```

Comment:
i would change resource to workload  and it would either have to be
`// ScaleDown downscales the workload` or `// ScaleDown scales down the workload`